test/unittests/libethereum/Block.cpp

```diff
@@ -108,10 +117,6 @@ BOOST_AUTO_TEST_CASE(bStates)
 	{
 		BOOST_ERROR("Failed test with Exception: " << _e.what());
 	}
-	catch(...)
```

Comment:
@winsvega  Don't  do this inside `BOOST_AUTO_TEST_CASE`. boost.test framework catches any uncaught exceptions and reports it itself.

`BOOST_REQUIRE_` etc. macros throw their internal exceptions themselves, they are caught is such `catch(...)` here and this leads to confusing error messages.